specification/iotsecurity/resource-manager/Microsoft.IoTSecurity/preview/2021-02-01-preview/devices.json

```diff
@@ -0,0 +1,746 @@
+          "type": "string",
+          "$ref": "#/definitions/DeviceDataSource",
+          "description": "Device data source."
+        },
+        "sensor": {
+          "readOnly": true,
+          "type": "object",
+          "description": "Sensor that scanned this device.",
+          "$ref": "#/definitions/Sensor"
+        },
+        "mergedDevices": {
+          "readOnly": true,
+          "type": "array",
+          "description": "List of merged devices data",
+          "items": {
+            "$ref": "#/definitions/MergedDevice"
+          }
+        },
+        "mergedToDeviceId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The device Id that this device was merged into"
+        },
+        "onboardingStatus": {
+          "readOnly": true,
+          "type": "string",
+          "enum": [
+            "Onboarded",
+            "NotOnboarded",
+            "NotSupported",
+            "InsufficientInfo"
+          ],
+          "x-ms-enum": {
+            "name": "OnboardingStatus",
+            "modelAsString": true
+          },
+          "description": "Device onboarding status."
+        },
+        "name": {
+          "type": "string",
+          "description": "Device name"
+        },
+        "deviceCategoryId": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 0,
+          "description": "Device category id"
+        },
+        "deviceTypeId": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 0,
+          "description": "Device type id"
+        },
+        "deviceSubTypeId": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 0,
+          "description": "Device sub type id"
+        },
+        "deviceCategoryDisplayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Device category display name"
+        },
+        "deviceTypeDisplayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Device type display name"
+        },
+        "deviceSubTypeDisplayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Device sub type display name"
+        },
+        "profilingConfidence": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "Confidence of the device profile"
+        },
+        "riskScore": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "risk score of the device."
+        },
+        "criticality": {
+          "type": "string",
+          "enum": [
+            "Low",
+            "Normal",
+            "High"
+          ],
+          "x-ms-enum": {
+            "name": "Criticality",
+            "modelAsString": true
+          },
+          "description": "Device criticality."
+        },
+        "tags": {
```

Comment:
Same comment as "name".  These are not ARM resource tags so they must be named differently

Reply:
Fixed